pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation-local.adoc

```diff
@@ -0,0 +1,247 @@
+
+The pods will take some time to start and must become ready before you can deploy the GKO (described in the next step).
+
+To monitor from the command line, run the following command:
+
+....
+kubectl wait --for=condition=ready pod -l app.kubernetes.io/name=apim3 --timeout 360s
+....
+
+After some time, the command-line output should look similar to the example below:
+
+....
+pod/apim-apim3-api-7b8b85d79-b7hkh condition met
+pod/apim-apim3-ui-5997c5d5c7-r9xdj condition met
+pod/apim-apim3-gateway-66cbd5c9d9-tv5jz condition met
+....
+
+You can increase the timeout if your machine is slower. If you encounter an error, you will not be able to proceed to the next steps - please contact Gravitee for assistance with this.
+
+NOTE: Alternatively, you can monitor your cluster and pods using link:https://k9scli.io/[K9s].
+
+
+== STEP 4: Check if the Console is up and running
+
+As noted in step 1 above, you can use your Console endpoint created with the cluster to check if the Console is up and running.
+
+To do so, open the Console endpoint URL in your browser - for example:
+
+http://localhost:9000/console/#!/login
+
+Log in the Console - the default local login credentials are `admin`/`admin`:
+
+image:{% link /images/apim/3.x/kubernetes/gko-deployment-local-console-login.png %}[]
+
+The Console dashboard should show one application running in the "Number of Applications" section of the Console dashboard:
+
+image:{% link /images/apim/3.x/kubernetes/gko-deployment-local-console-1.png %}[]
+
+
+== STEP 5: Deploy the GKO on the cluster
+
+Run the following command in your command-line tool (the working directory does not matter) to deploy the GKO on the cluster of your current Kubernetes context:
+
+....
+kubectl apply -f https://github.com/gravitee-io/gravitee-kubernetes-operator/releases/download/0.1.0-alpha.2/bundle.yml
+....
+
+The operation is quick and the successful command-line output should be similar to one in the example below:
+
+....
+namespace/gko-system created
+customresourcedefinition.apiextensions.k8s.io/apidefinitions.gravitee.io created
+customresourcedefinition.apiextensions.k8s.io/managementcontexts.gravitee.io created
+serviceaccount/gko-controller-manager created
+role.rbac.authorization.k8s.io/gko-leader-election-role created
+clusterrole.rbac.authorization.k8s.io/gko-manager-role created
+clusterrole.rbac.authorization.k8s.io/gko-metrics-reader created
+clusterrole.rbac.authorization.k8s.io/gko-proxy-role created
+rolebinding.rbac.authorization.k8s.io/gko-leader-election-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/gko-manager-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/gko-proxy-rolebinding created
+configmap/gko-manager-config created
+service/gko-controller-manager-metrics-service created
+deployment.apps/gko-controller-manager created
+....
+
+The GKO has now been deployed on your local cluster.
+
+
+== STEP 6: Create a Management Context custom resource
```

Comment:
the following steps are not necessary for the installation (only). but more a demonstration. or a predefined example

Reply:
All relevant sections fully refactored now... this + ApiDefinition + API Call steps are now part of a separate section for playing/trying out the feature.